src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/logs-highlightsearch.pipe.ts

```diff
@@ -0,0 +1,26 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'highlightSearch'
+})
+export class LogsHighlightSearchPipe implements PipeTransform {
```

Comment:
? if yes, then file name will be search-highlight.pipe.ts. WDYT?
```suggestion
export class SearchHighlightPipe implements PipeTransform {
```

Reply:
`Logs` was added as its not generic (as of now) and specific to cluster logs. If there is possibility to be generic, sure that makes sense. But this makes me wonder assuming this file is specific to cluster logs, is it better to keep it in `ceph-> cluster-> logs` (with the log components) or just `ceph-> cluster` (current location). WDYT?

Reply:
let's make this pipe generic and put it in the pipes.module. Maybe we can use it in some place in the future.

Reply:
Done.